src/dndbeyond/content-scripts/character.js

```diff
@@ -873,24 +873,79 @@ function displayAction(paneClass) {
     });
 }
 
+function handleCustomText(paneClass) {
+    // Function Objective: Search for and handle block of VTT Message text...
+    //...in pane Notes or Description
+    const outCustomRollFuncs = {
+        before: [],
+        replace:[],
+        after:  [],
+        roll: (customTextFuncList, normalRollFunction=undefined) => {
+            if (customTextFuncList.length > 0) {
+                for (let i = 0; i < customTextFuncList.length; i++) {
+                    customTextFuncList[i]();
+                }
+            } else if (normalRollFunction) {
+                normalRollFunction();
+            }
+        },
+    };
+    const regexp = /```\[([^\]]*)\]\s*((\S|\s)*?)\s*```/g;
+    const rollOrderTypes = ["before", "after", "replace"];//(relative to normal roll msg)
+    const paneText = descriptionToString($(`.${paneClass}`).find(".ddbc-property-list__property:contains('Notes:'), .ct-action-detail__description, .ct-spell-detail__description, .ct-item-detail__description, .ddbc-action-detail__description, .ddbc-spell-detail__description, .ddbc-item-detail__description"));
+    console.log("Pane Text: " + paneText);
+    const matchedText = [...paneText.matchAll(regexp)];
+    //Handle Matched Text
+    if (matchedText) {
```

Comment:
`matchedText` is the array you defined above so it will always evaluate to true. You'd  want to test for `matchedText.length`